docs/contribute/doc.md

```diff
@@ -130,12 +130,14 @@ doc entry to the appropriate page in the online docs.
 
 Example:
 ```lean
+/--
+describe what the command does here
+-/
 add_tactic_doc
 { name := "display name of the tactic",
   category := cat,
   decl_names := [`dcl_1, `dcl_2],
-  tags := ["tag_1", "tag_2"],
-  description := "describe what the command does here"
+  tags := ["tag_1", "tag_2"]
 }
 ```
 
```

Comment:
I can't seem to comment / give a suggestion on the exact line itself, but `description` is mentioned below and should be updated per the change to the docstring of `add_tactic_doc_command`. 

Otherwise, LGTM!

Reply:
Actually, I was going to make this change myself but then I got confused by the wording here:

> If only one related declaration is listed in `decl_names` and it does not have a doc string, `description` will be automatically added as its doc string.

My initial reading of this is that given:
```lean
/-- stuff -/
add_tactic_doc
{ name := "display name of the tactic",
  category := cat,
  decl_names := [`dcl_1],
  tags := ["tag_1", "tag_2"],
}
```
If `dcl_1` does not have a doc string, then `/-- stuff -/` will be added as the doc string of `dcl_1`. But it's actually the other way around, right? That is, if the `/-- stuff -/` were missing, then this doc entry would inherit the doc string of `dcl_1` (and throw an error if `dcl_1` had no doc string).

I suggest changing that text (here and in the relevant parts in `tactic.doc_commands` to something like:

> If only one related declaration is listed in `decl_names` and if this invocation of `add_tactic_doc` does not have a doc string, the doc string of that declaration will become the body of the tactic doc entry.